cmd/util/cmd/addresses/cmd.go

```diff
@@ -0,0 +1,54 @@
+package addresses
+
+import (
+	"github.com/rs/zerolog/log"
+	"github.com/spf13/cobra"
+
+	"github.com/onflow/flow-go/model/flow"
+)
+
+var (
+	flagChain          string
+	flagCount          int
+	flagCommaSeparated bool
```

Comment:
Suggestion: 

`flagCommaSeparated` ->  `flagSeparator`

so user can specify any separator.

Reply:
Good idea! Changed in d489273c82cb0b424c9372de1859464fe7bf44f7